.circleci/config.yml

```diff
@@ -1036,6 +1024,81 @@ jobs:
     steps:
       - run: "echo ok"
 
+  show-coverage:
+    docker:
+      - image: *GOLANG_IMAGE
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: collate coverage output into single file and convert to readable format
+          command: |
+            mkdir -p /tmp/output
+            echo "mode: set" > cov.txt
+            for f in /tmp/coverage/*; do tail -n +2 $f >> cov.txt; done
+            go tool cover -func=cov.txt -o /tmp/output/coverage_by_function.txt
+            cp cov.txt /tmp/output/cov.txt
+      - run: sudo apt-get install jq
+      - run:
+          name: get diff of test coverage from changes
+          command: |
+            file_url_response=$(curl --location --request GET "https://api.github.com/repos/hashicorp/consul/contents/.github/cover.out?ref=$TEST_COVERAGE_GOLDEN_BRANCH" \
```

Comment:
I think it's better to extract the URL into a variable to make it easier to port to enterprise.

Reply:
extracted project into variable for ease of porting